src/item.cpp

```diff
@@ -1889,13 +1889,15 @@ void item::gunmod_info( std::vector<iteminfo> &info, const iteminfo_query *parts
                                       at->name() ) ) );
         }
     }
-    if( mod.reload_modifier != 0 && parts->test( iteminfo_parts::GUNMOD_RELOAD ) ) {
+    if( mod.reload_modifier != 1 && parts->test( iteminfo_parts::GUNMOD_RELOAD ) ) {
         info.emplace_back( "GUNMOD", _( "Reload modifier: " ), _( "<num>%" ),
-                           iteminfo::lower_is_better, mod.reload_modifier );
+                           iteminfo::lower_is_better | iteminfo::is_decimal | iteminfo::lower_is_better, 
```

Comment:
```suggestion
                           iteminfo::lower_is_better | iteminfo::is_decimal,
```

Reply:
Oh crap, that was supposed to be `no_newline`, thanks for spotting that